src/taipy/core/config/checkers/_data_node_config_checker.py

```diff
@@ -94,28 +94,45 @@ def _check_required_properties(self, data_node_config_id: str, data_node_config:
                                 f"property `{required_property}` for type `{storage_type}`.",
                             )
 
-    def _check_generic_read_write_fct_params(self, data_node_config_id: str, data_node_config: DataNodeConfig):
+    def _check_generic_read_write_fct_and_args(self, data_node_config_id: str, data_node_config: DataNodeConfig):
         if data_node_config.storage_type == DataNodeConfig._STORAGE_TYPE_VALUE_GENERIC:
             properties_to_check = [
-                DataNodeConfig._OPTIONAL_READ_FUNCTION_PARAMS_GENERIC_PROPERTY,
-                DataNodeConfig._OPTIONAL_WRITE_FUNCTION_PARAMS_GENERIC_PROPERTY,
+                DataNodeConfig._OPTIONAL_READ_FUNCTION_ARGS_GENERIC_PROPERTY,
+                DataNodeConfig._OPTIONAL_WRITE_FUNCTION_ARGS_GENERIC_PROPERTY,
             ]
             for prop_key in properties_to_check:
                 if data_node_config.properties and prop_key in data_node_config.properties:
                     prop_value = data_node_config.properties[prop_key]
-                    if not isinstance(prop_value, list):
+                    if not isinstance(prop_value, (list, tuple)):
```

Comment:
On the documentation, it is said we support the list only. Is there a specific reason to add a tuple here?

Reply:
no particular reason, I just think it provides user with more option for input type :D if it doesn't make sense to you, we can move it back to list only :D

Reply:
Not a strong opinion on it. Keeping a single way (only list) makes it easier to document, but I am fine with keeping tuples as well. It s up to you.

Reply:
Hmm I agree and I think it's something that developers would also expect as a default behavior also. I'll revert it back to only list then. Thanks!